napari/_qt/qt_event_loop.py

```diff
@@ -416,7 +368,7 @@ def run(
         has at least ``max_loop_level`` event loops running.  By default, 1.
     _func_name : str, optional
         name of calling function, by default 'run'.  This is only here to
-        provide functions like `gui_qt` a way to inject their name into the
+        provide functions like a way to inject their name into the
```

Comment:
Something is missing here...
Should the whole kwarg be removed?

Reply:
I think it is ok to leave. If there is a deprecation in the future that touches the event loop then it would make sense to have it available.

I did fix the wording.